packages/back-end/src/services/features.ts

```diff
@@ -120,17 +135,24 @@ function generateVisualExperimentsPayload({
         phase?.savedGroups
       );
 
+      const urlRedirects = v.urlRedirects;
+
       if (!phase) return null;
 
       const exp: AutoExperimentWithProject = {
         key: e.trackingKey,
         status: e.status,
         project: e.project,
-        variations: v.visualChanges.map((vc) => ({
-          css: vc.css,
-          js: vc.js || "",
-          domMutations: vc.domMutations,
-        })) as AutoExperimentWithProject["variations"],
+        variations: urlRedirects
+          ? (urlRedirects.map((r) => ({
+              urlRedirect: r.url,
+            })) as AutoExperimentWithProject["variations"])
+          : (v.visualChanges.map((vc) => ({
+              css: vc.css,
+              js: vc.js || "",
+              domMutations: vc.domMutations,
+            })) as AutoExperimentWithProject["variations"]),
+        persistQueryString: v.persistQueryString,
```

Comment:
don't ship this down if undefined. Conditionally add this if a url redirect experiment